### DISCUSSION TITLE
[@types/three] dolly functions missing

### DISCUSSION THREAD

**Discussion by sancelot** [DefinitelyTyped/DefinitelyTyped#57054] [Issues with a @types package]
dollyIn() / dollyOut() functions are missing in orbitcontrol 

/examples/jsm/controls/OrbitControls.d.ts

[a comment by an automated account is not shown]

**Comment by joshuaellis** [comment]
We have a repo over at three-TS-types, please make an issue there, or better please submit a PR to resolve the issue ☺️
